src/components/Form.js

```diff
@@ -22,12 +22,11 @@ function Form() {
 
   return (
     <>
+      <h2 className="form-title">Add New Books</h2>
       <form onSubmit={handleSubmit}>
-        <input type="text" placeholder="Please enter title" value={title} onChange={(e) => setTitle(e.target.value)} required />
-        <br />
-        <input type="text" placeholder="Please select categories" value={category} onChange={(e) => setCategory(e.target.value)} required />
-        <br />
-        <button type="submit">Submit</button>
+        <input className="title" type="text" placeholder="Please enter title" value={title} onChange={(e) => setTitle(e.target.value)} required />
+        <input className="category" type="text" placeholder="Please select categories" value={category} onChange={(e) => setCategory(e.target.value)} required />
+        <button type="submit" className="btn-submit">ADD BOOK</button>
       </form>
     </>
   );
```

Comment:
- [ ] Kindly, compare your implementation with the design provided to meet all the requirements.

Original design

![header--2](https://user-images.githubusercontent.com/55635977/150628539-42beaee7-f31b-4e70-83f7-59b1b7c3a339.png)

Your design

![header--4](https://user-images.githubusercontent.com/55635977/150628544-9e4ff05d-0200-47f8-80dd-fb3143d0a1af.png)